server/data/prisonerOffenderSearchTypes.ts

```diff
@@ -3,3 +3,5 @@ import { components } from '../@types/prisoner-offender-search-api'
 export type CurrentIncentive = components['schemas']['CurrentIncentive']
 
 export type Prisoner = components['schemas']['Prisoner']
+export type PagePrisoner = components['schemas']['PagePrisoner']
+export type PaginationRequest = components['schemas']['PaginationRequest']
```

Comment:
Using standard types for results from prisoner-offender-search rather than anonymous objects to represent them.